tasks/main.yml

```diff
@@ -25,6 +33,14 @@
 - name: Create data directory
   file: path={{ otp_data_dir }} mode=0775 state=directory
 
-- name: Install upstart service
-  template: src=otp.conf.j2 dest=/etc/init/otp.conf
+- name: Install service config
+  template: src=otp.service.j2 dest=/lib/systemd/system/otp.service
   notify: Restart OpenTripPlanner
+  register: otp_service_config
+
+  # We need to reload systemd config for it to see changes.  This (and the 'register' just above)
+  # will not be necessary in Ansible 2.2 because there will be a 'systemd' module with a
+  # 'daemon_reload' option that we can use instead of 'service'.
+- name: Notify systemd of service config changes
```

Comment:
That module is new for Ansible 2.2.  I got the workaround from https://github.com/ansible/ansible-modules-core/issues/191, which is marked as resolved by https://github.com/ansible/ansible-modules-core/pull/3660, the creation of the systemd module.